core/common/src/test/java/tachyon/wire/BlockInfoTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.wire;
+
+import java.util.List;
+import java.util.Random;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.common.collect.Lists;
+
+public class BlockInfoTest {
```

Comment:
By default, JSON serialization uses auto-detection based on the getter/setters, which means the mapper will call the getter/setters. http://www.cowtowncoder.com/blog/archives/2011/02/entry_443.html

 However, since you're using @JsonProperty here to force explicit annotations on fields, perhaps  you can change the visibility for auto-detection during tests, as in http://stackoverflow.com/questions/7105745/how-to-specify-jackson-to-only-use-fields-preferably-globally.

Lastly, since we already have all the getter/setters, @JsonProperty seems not necessary any more.